style.css

```diff
@@ -75,6 +75,112 @@ body {
   color: #3a4a42;
 }
 
+.popup_container {
+  width: 100vw;
+  height: 100vh;
+  background: white;
+  padding: 0.5rem;
+  position: fixed;
+  display: none;
+  top: 0;
+  left: 0;
+  z-index: 20;
+  overflow: scroll;
+}
+
+.popup_mobile {
+  display: inline-block;
+  width: 90%;
+}
+
+.popup_desktop {
+  display: none;
+}
+
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Make sure the max width of your popup windows is 920px as shown on the Figma design

Your Implementation | Figma Design
:---:|:---:
![181](https://user-images.githubusercontent.com/79058364/189106792-88e6a0ec-99f7-4c4e-923e-759a412258dc.png) | ![181](https://user-images.githubusercontent.com/79058364/189106086-ad46721f-211a-41e9-97b7-421fb32231c2.png) 


- [ ]  Please do the same for the desktop size 👨🏻‍🎨 🎨 

- Actually, I don't see the full height of your page popups when I'm on the desktop size, please make sure 100% height is shown 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![18](https://user-images.githubusercontent.com/79058364/189106882-d3760ed4-8fd8-4538-ad6e-184bffdf8946.png) | ![18](https://user-images.githubusercontent.com/79058364/189106145-f947cebe-3a5a-45a8-9851-ada6c6544c46.png)

Reply:
I don't know why the project still scroll beneath the pop up window, I have used the overflow: hidden but it still scroll